src/elements/Accordion/Accordion.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Accordion
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+import Grid from '../Grid/Grid';
+import { isMobile } from '../../util/helpers';
+
+class Accordion extends Component {
+	static propTypes = {
+		children: PropTypes.node.isRequired,
+		title: PropTypes.node.isRequired,
+		className: PropTypes.string,
+		expanded: PropTypes.bool,
+		notification: PropTypes.bool // this is used to style accordion as a notification
+	};
+
+	constructor(props) {
+		super(props);
+
+		if (this.props.expanded) {
+			this.state = {
+				showContent: true
+			};
+		} else {
+			this.state = {
+				showContent: false
+			};
+		}
+
+		this.toggleAccordion = this.toggleAccordion.bind(this);
+	}
+
+	toggleAccordion() {
+		this.setState({
+			showContent: !this.state.showContent
+		});
+	}
+
+	render() {
+		const { title, children, className, notification } = this.props,
+			{ showContent } = this.state,
+			itemWidths = showContent && !isMobile() && notification ? [3, 7, 2] : [10, 2],
+			toggleText = showContent ? 'hide' : 'show',
+			toggleIconClass = showContent ? 'arrow arrow-up' : 'arrow arrow-down';
+
+		const combinedClasses = classNames(
+			'ui-accordion',
+			this.props.notification && 'notification',
+			className
+		);
+
+		return (
+			<Div className={combinedClasses}>
+				<Grid
+					gutter="xx-small"
+					itemWidths={itemWidths}
+				>
+					<Div className="title">
+						{title}
+					</Div>
+					{showContent && !isMobile() && notification &&
+						<Div>{children}</Div>
+					}
+					<Div onClick={this.toggleAccordion} right className="ui-accordion-toggle">
+						<Grid gutter="xx-small" itemWidths={[10, 2]}>
+							<Div>{toggleText}</Div>
+							<Div><Div className={toggleIconClass}>^</Div></Div>
+						</Grid>
+					</Div>
+				</Grid>
+				{showContent && (isMobile() || !notification) &&
+					<Div className="accordion-content">{children}</Div>
```

Comment:
Same thing, since this is just a regular div with no need for properties, and it's not being used a child in the Grid, it should be a just a regular div.